CMakeLists.txt

```diff
@@ -8,7 +8,8 @@ project(clio VERSION ${CLIO_VERSION} HOMEPAGE_URL "https://github.com/XRPLF/clio
 
 # =========================== Options ====================================== #
 option(verbose "Verbose build" FALSE)
-option(tests "Build tests" FALSE)
+option(tests "Build unit tests" FALSE)
+option(integration_tests "Build database tests" FALSE)
```

Comment:
I think here and in conanfile these should be `true` by default.

Reply:
I don't think so.. why would it be? 
Normal use-case for non-clio-devs is to build Clio for running it.. why would they need to mess around with unit and integration tests? It was false all along for this reason.